mino/minogrpc/mod.go

```diff
@@ -143,3 +147,10 @@ func (m Minogrpc) MakeRPC(name string, h mino.Handler) (mino.RPC, error) {
 
 	return rpc, nil
 }
+
+// AddNeighbours fills the neighbours map of the server
+func (m Minogrpc) AddNeighbours(minoGrpcs ...*Minogrpc) {
```

Comment:
You'll have to change this anyway when going to real world, so maybe it's the right time to make an API to pass the certificate?